src/ui.js

```diff
@@ -12,8 +12,9 @@ export default class Ui {
    * @param {object} ui.api - Editor.js API
    * @param {ImageConfig} ui.config - user config
    * @param {Function} ui.onSelectFile - callback for clicks on Select file button
+   * @param {boolean} ui.readOnly - should draw UI in read-only mode
    */
-  constructor({ api, config, onSelectFile }) {
+  constructor({ api, config, onSelectFile, readOnly }) {
```

Comment:
the `readOnly` is not relevant to this PR

Reply:
And I didn't write it, its from auto merge. What should I do about it ?

Reply:
@MohitKS5 to resolve the above problem:
Please Clone the Master branch and make your changes and then commit.
I guess You had clone read-only branch which is the reason why the above changes are there.

Reply:
yeah, I know why this happened and I mentioned it in the first commit of PR itself that I don't think this would be merged because its based on read-only branch. Then I just followed what reviewers asked, It could have been avoided with rebase instead of pull but I doubt we can go back now. Anyways, I did a rebase now and commits tab is much cleaner but the commit titled "initial" still remains and says authored by me. I have checked "allow edit by maintainers" for this PR. Feel free to do changes or close the PR.

Reply:
@robonetphy @neSpecc I used  `rebase onto` and removed that commit.